src/opm/parser/eclipse/EclipseState/AquiferCT.cpp

```diff
@@ -31,17 +31,26 @@ namespace Opm {
         for (auto& aquctRecord : aquctKeyword){
   
             AquiferCT::AQUCT_data data;
-    
+          
+            if( deck.hasKeyword( "FIELD" ) ) {
```

Comment:
OK; the best solution here would be to invoke the unit system to perform a numerical conversion on your behalf, but I realize that will require some code changes in the unit system. This should be an OK solution:

const auto& unitSystem = deck.getUnitSystem();
switch( unitSystem.getType() ) {
     case FIELD:
         c1 = ..
         c2 = ...
         break;
      case METRIC:
         c1 = 
         c2 = 
         break
     LAB:
     
..